docs/HowToRunSTDKSample.md

```diff
@@ -0,0 +1,254 @@
+        - Create a new profile to add it to project.   
+        Fill properties of device and select 'NEXT'.   
+![Select create a new one.](./media/STDK_create_a_new_profile_2.png)   
+        - Add copabilities for device.   
+        Select 'ADD CAPABILITY'. And add Capabilities like switch, switch level, color control and health check. 
+![Select create a new one.](./media/STDK_Add_Copability_1.png)   
+        At this time, please note that you will need to add a "Health Check" capability to update the connectivity status of a device. It is only required on the main component.   
+![Select create a new one.](./media/STDK_Add_Copability_2.png)   
+        - Select UI Display.   
+        Select Main State and Action.   
+![Select create a new one.](./media/STDK_Select_UI_Display.png)   
+        - Add Device Onboarding.   
+        This defines information to support the initial connection process between IoT device and SmartThings Cloud.   
+        Click the "GO TO DEVICE ONBOARDING" and then, enter the all remaining information required.   
+![ADD DEVICE ONBOARDING](./media/STDK_Add_Device_Onboarding_1.png)   
+![ADD DEVICE ONBOARDING](./media/STDK_Add_Device_Onboarding_2.png)   
+![ADD DEVICE ONBOARDING](./media/STDK_Add_Device_Onboarding_3.png)   
+        - Deploy Device Profile to test.   
+        You can publish your device to the SmartThings platform for testing. And then you will be able to access your device through the SmartThings app. Actually this step is for self-testing. If you want to officially publish your device with enrolled organizations ID(e.g. company MNID), please refer to the process below.   
+![ADD DEVICE ONBOARDING](./media/STDK_Deploy_1.png)   
+        - Register Device Identity.   
+        This step uploads device identity data generated in the Create device identity phase.
+![ADD DEVICE ONBOARDING](./media/STDK_Deploy_2.png)   
+        Paste the public key that copied key in the Public Key value box that appears. And then, click ADD button.
+![ADD DEVICE ONBOARDING](./media/STDK_Input_Public_Key.png)   
+
+### Build and Flash   
+#### Put device information into sample application folder.   
+1. Download onboarding data.   
+Download "onboarding_config.json" file into "[TizenRT]/apps/examples/stdk_smart_lamp/"   
+![ADD DEVICE ONBOARDING](./media/STDK_Download_onboarding_config.png)   
+1. Copy Device Identity file into application folder.   
+```bash   
+// in [TizenRT]/external/stdk/st-device-sdk-c/tools/keygen/linux
+$ cd external/stdk/st-device-sdk-c/tools/keygen/linux
+
+$ cat output_STDK****ce**2**3/device_info.json   
+{
+  "deviceInfo": {
+    "firmwareVersion": "V201910",
+    "privateKey": "3d********AOyY********ezJQ********TMKLGxzbQ=",
+    "publicKey": "1D********a21F********8WwP********yU/n8vFvM=",
+    "serialNumber": "STDK****ce**2**3"
+  }
+}
+
+$ cp ./output_STDK****ce**2**3/device_info.json ../../../../../../apps/examples/stdk_smart_lamp/apps/examples/stdk_smart_lamp/   
```

Comment:
The relative path looks somewhat messy, Maybe performing the cp from TizenRT root folder would be better. Please ignore the suggestion if you feel unecessary,

Reply:
Has added an explains.